backend/routes/offline-payment.js

```diff
@@ -8,6 +8,38 @@ const { getConfig } = require('../utils/encryptedConfig')
 const makeOffer = require('./_makeOffer')
 
 module.exports = function (router) {
+  router.post(
```

Comment:
nit: the naming makes it pretty obvious but perhaps still document this route to explain when it gets called by the UI